src/coreclr/tools/SuperFileCheck/Program.cs

```diff
@@ -100,13 +100,13 @@ static void VerifyCheckPrefixes(string str, string[] checkPrefixes)
         /// </summary>
         static async Task<FileCheckResult> RunLLVMFileCheckAsync(string[] args)
         {
-            var startInfo = new ProcessStartInfo();
-            startInfo.FileName = FileCheckPath;
-            startInfo.Arguments = String.Join(' ', args);
-            startInfo.CreateNoWindow = true;
-            startInfo.WindowStyle = ProcessWindowStyle.Hidden;
-            startInfo.RedirectStandardOutput = true;
-            startInfo.RedirectStandardError = true;
+            var startInfo = new ProcessStartInfo(FileCheckPath, args)
```

Comment:
I am afraid that we need to revert the changes to this file and `src/mono/wasm/debugger/DebuggerTestSuite/TestHarnessStartup.cs`, for some reason they are not built using the local build of dotnet/runtime but the pre-downloaded SDK (perhaps they are required to build the repo itself or are just targeting older monikers for some reason).